### DISCUSSION TITLE
Blueprints - Selectors should pre-select item if only one match

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#7998] [Automations / scripts / scenes] [answered]
Most Blueprint selectors filter by model and manufacturer, which is great - it means the user doesn't have to browse through a lengthy list and can quickly make a selection. Really convenient. 

But that got me thinking, if there is only ONE match, why not just select that by default? It would save two clicks and be even more convenient.

![image](https://i.imgur.com/ag30eeJ.png)

**Comment by frenck** [comment, marked as answer]
For me personally, generally don't like that.

While, in general, this is ease of use, it also adds to the ease of error. I think one should explicitly make a choice as default values are easily oversight (and could trigger... 🤷  You know what?! The fire extinguisher system, self-destruct sequences, or anything less of an unwished, less destructive, thing).
